[2020-06-03 14:54:22,018] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: covid_tasks.get_covid_data_task 2020-06-03T09:23:52.816463+00:00 [queued]>
[2020-06-03 14:54:22,025] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: covid_tasks.get_covid_data_task 2020-06-03T09:23:52.816463+00:00 [queued]>
[2020-06-03 14:54:22,025] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-03 14:54:22,025] {taskinstance.py:880} INFO - Starting attempt 1 of 2
[2020-06-03 14:54:22,025] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-03 14:54:22,120] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): get_covid_data_task> on 2020-06-03T09:23:52.816463+00:00
[2020-06-03 14:54:22,335] {standard_task_runner.py:53} INFO - Started process 247 to run task
[2020-06-03 14:54:22,695] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: covid_tasks.get_covid_data_task 2020-06-03T09:23:52.816463+00:00 [running]> LAPTOP-P8ICNHVB.localdomain
[2020-06-03 14:54:23,100] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-06-03 14:54:23,104] {taskinstance.py:1052} INFO - Marking task as SUCCESS.dag_id=covid_tasks, task_id=get_covid_data_task, execution_date=20200603T092352, start_date=20200603T092422, end_date=20200603T092423
[2020-06-03 14:54:31,927] {logging_mixin.py:112} INFO - [2020-06-03 14:54:31,927] {local_task_job.py:103} INFO - Task exited with return code 0
